build-scripts/profile_tool.py

```diff
@@ -121,8 +122,10 @@ def main():
         subtracted_profile = profile1 - profile2
 
         exclusive_rules = len(subtracted_profile.get_rule_selectors())
-        if exclusive_rules > 0:
+        exclusive_vars = len(subtracted_profile.get_variable_selectors())
+        if exclusive_rules > 0 or exclusive_vars > 0:
```

Comment:
Please make two separate if statements, because if one of them is zero it will be printed for example: `0 rules were left...`